src/core/batch/batch.test.ts

```diff
@@ -1,7 +1,7 @@
-import { batchIncludes } from "./batch";
+import { includes } from "./batch";
```

Comment:
was this an eslint recommendation to avoid filename "stuttering"?

Reply:
No, we said we wanted to change all methods names to start with a verb. This seemed appropriate since the module was already named "batch." Do you have an alternate suggestion?

Reply:
Nah, I was just wondering what the impetus was; I like it, but  the verb thing hadn't occurred to me.